modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowService.java

```diff
@@ -229,7 +231,7 @@ WorkflowInstance start(WorkflowDefinition workflowDefinition, MediaPackage media
    * @throws WorkflowStateException
    *           if the workflow instance is in a disallowed state
    */
-  void remove(long workflowInstanceId) throws WorkflowDatabaseException, WorkflowParsingException, NotFoundException,
+  void remove(long workflowInstanceId, boolean force) throws WorkflowDatabaseException, WorkflowParsingException, NotFoundException,
```

Comment:
Instead of having to define the default (`false`) over and over again, would it make sense to have ` void remove(long workflowInstanceId)` and ` void remove(long workflowInstanceId, boolean force)` instead, with the former defined as

```java
 void remove(long workflowInstanceId … {
  remove(workflowInstanceId, false);
}
```

Reply:
You're right, done! I also extended one test a bit.